apps/core/components/userContext/Firebase.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import firebase from 'firebase/app';
```

Comment:
In my other project, I only have `import firebase from '@firebase/app';` instead of the two imports, and note as well the @ prefix. Not sure how relevant it is.

Reply:
But just `import firebase from 'firebase/app';` is not enough. If I want to use `auth`, I need to import it as well.

The other option is to use `import * as firebase from 'firebase';` and import everything, but it is not recommended and you'll get console warning:
`It looks like you're using the development build of the Firebase JS SDK.
When deploying Firebase apps to production, it is advisable to only import
the individual SDK components you intend to use.`

And what exactly you mean by `@ prefix`? I don't see any `@` laying around this file

Reply:
>  If I want to use auth, I need to import it as well.
I think you need to import it in the files where you use `firebase.auth(). ...`, but in here you're only initializing the app, so I'm not sure it's necessary.

The `@` prefix was actually a mistake on my part which I will need to fix in my project, the `@firebase/app` package is not meant to be used on its own. 

>This package is not intended for direct usage, and should only be used via the officially supported firebase package.

cf https://www.npmjs.com/package/@firebase/app